assets/stylesheets/kitten/utilities/display/_hidden.scss

```diff
@@ -0,0 +1,39 @@
+/// Utility to hide an item according to the media-query.
```

Comment:
J'aurais mis du pluriel partout `Utilities to hide items according to media-queries.`, non ?